challenges/arrayReverse/arrayShift/array-shift.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+
+function insertShiftArray(arr, num){
+    var toogle = true;
+var newArray = [];
+    for(var i = 0; i < arr.length; i++)
+    {
+        if(arr[i] < num)
+        {
+          newArray.push(arr[i]);  
+        }if(arr[i] > num && toogle === true ){
+            newArray.push(num);
+           toogle = false
+        }if(arr[i] > num && toogle === false){
+          newArray.push(arr[i]);
+        }
+
+        
+    }
```

Comment:
Looks really great, there is a way to do it without .push() that would be great knowledge to have!